package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-hook-form-jsonschema",
-  "version": "0.1.3",
+  "version": "0.2.0-beta.0",
```

Comment:
🤔

Reply:
I adopted a beta relase so I can do breaking changes for now without needing to up the minor or major